tox.ini

```diff
@@ -17,7 +17,7 @@ whitelist_externals = /bin/bash
 commands =
     {[base]commands}
     py27: py.test -v --cov portal --cov-report xml:"{toxinidir}/coverage.xml" []
-    py34: py.test -v tests/test_dict_tools.py tests/test_date_tools.py tests/test_access_on_verify.py tests/test_access_url.py tests/test_address.py tests/test_app_text.py tests/test_assessment_engine.py tests/test_assessment_status.py tests/test_audit.py tests/test_auth.py []
+    py34: py.test -v tests/test_dict_tools.py tests/test_date_tools.py tests/test_access_on_verify.py tests/test_access_url.py tests/test_address.py tests/test_app_text.py tests/test_assessment_engine.py tests/test_assessment_status.py tests/test_audit.py tests/test_auth.py tests/test_clinical.py[]
```

Comment:
Could you add the space back? `[]` is a placeholder for extra arguments to `py.test` and without it you would need to pass an extra space every time.